Description: Prefer X11 session over wayland
 As we are not ready yet for switching to wayland, let's prefer X11
 session when no specific choice was made.
 This will be reverted once we switch to wayland by default.
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1703601
Forwarded: not-needed
Author: Didier Roche <didrocks@ubuntu.com>

Index: gdm3/daemon/gdm-session.c
===================================================================
--- gdm3.orig/daemon/gdm-session.c
+++ gdm3/daemon/gdm-session.c
@@ -359,11 +359,7 @@ get_system_session_dirs (GdmSession *sel
 
 #ifdef ENABLE_WAYLAND_SUPPORT
         if (!self->priv->ignore_wayland) {
-#ifdef ENABLE_USER_DISPLAY_SERVER
-                g_array_prepend_val (search_array, wayland_search_dir);
-#else
                 g_array_append_val (search_array, wayland_search_dir);
-#endif
         }
 #endif
 
